tests/scripts/change_mac.sh

```diff
@@ -2,12 +2,14 @@
 
```

Comment:
We already have one script already
ansible/roles/test/files/helpers/change_mac.sh
Why do we need another one?

Reply:
Is it possible to create one common file and use it on both places?

Reply:
Yes it is possible, but I am not sure if it is a good choice.

Currently in sonic-mgmt repo, there are two automation strategies: 
* ansible playbook: scripts under sonic-mgmt/ansible
* pytest: scripts under sonic-mgmt/tests

In case pytest scripts need to re-use helper scripts under sonic-mgmt/ansible, there are 3 options:
1. Directly call the helper scripts under sonic-mgmt/ansible. This is the idea you suggested: "create one common file and use it on both places". Then in pytest scripts under sonic-mgmt/tests, code may look like below:
```
ptfhost.script("../ansible/roles/test/files/helpers/change_mac.sh")
```
2. Copy the helper scripts to `sonic-mgmt/tests/scripts`. This is the option I used in the first commit of this PR.
3. Create symbolic links to `sonic-mgmt/ansible/roles/test/files/helpers` under `sonic-mgmt/tests/scripts`. This is the option I used in the last commit of this PR.

If the long term goal is to implement new automation using pytest and gradually deprecate the ansible playbook scripts, then personally I prefer the third option. Please advice which option you prefer. Or you have other better ideas?